README.md

```diff
@@ -1,3 +1,59 @@
-[![Telegraph Logo](https://github.com/knocklabs/telegraph/assets/29106675/9216da6c-5a64-4bc5-88dc-00a905b04bfc)](https://knock.app)
+![Telegraph by Knock](https://github.com/knocklabs/telegraph/assets/29106675/11a9e54e-2388-4c5d-9769-16099c5de9dc)
```

Comment:
Uber nit: I think we should make this smaller vertically. It feels too large especially when the banner is on a short readme